src/styles/blocks/style-block-review.css

```diff
@@ -0,0 +1,54 @@
+.review {
+  display: flex;
+  justify-content: center;
+}
+
+.review--size {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: flex-end;
+  gap: 19px;
+  margin: 0 16px 24px;
```

Comment:
redundant modifier and redundant BEM block, according to the task:
![image](https://github.com/mate-academy/layout_product-cards/assets/80700247/7c27f6c9-4ab1-490a-8eca-ebc07fc212c3)